lib/index.js

```diff
@@ -20,20 +22,20 @@ gulpPep8.prototype.lint = (file, opts) => {
 
     for (k in opts) {
       // allow custom lint, also used when testing
-      if (k=='lintFunc'){
+      if (k === 'lintFunc') {
```

Comment:
Added space and `===` but `===` is not super necessary here, and in the other places. It's just a good practice because of the type coercing in JS especially for `null-ish values`.